### ISSUE TITLE
Fail to select PrestoSQL UUID value while connecting with PrestoDB Driver Manager

### ISSUE THREAD

**Issue by guyco33** [dbeaver/dbeaver#6270]
#### System information: 
- Operating system (distribution) and version 
- DBeaver version 6.1.2
- Java version OpenJDK 11

#### Connection specification:
- PrestoSQL version 316 (https://prestosql.io/docs/current/language/types.html)
- PrestoDB Facebook version 0.221

#### Problem description:
Fail to select PrestoSQL UUID value while connecting with PrestoDB Driver Manager

#### Errors/backtraces from the logs

```
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error: Error executing query
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:134)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:467)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$0(SQLQueryJob.java:407)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:146)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:405)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:849)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:2778)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:98)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:146)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:96)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:102)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: Error executing query
	at com.facebook.presto.jdbc.PrestoStatement.internalExecute(PrestoStatement.java:279)
	at com.facebook.presto.jdbc.PrestoStatement.execute(PrestoStatement.java:228)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:338)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 11 more
Caused by: java.lang.RuntimeException: Error fetching next at https://presto-test.gett.io:8443/v1/statement/executing/20190712_134639_00013_8nfe8/x5ab6b8afff9f44a284f0fc7ae99013b6/0 returned an invalid response: JsonResponse{statusCode=200, statusMessage=OK, headers={content-type=[application/json], date=[Fri, 12 Jul 2019 13:46:40 GMT], vary=[Accept-Encoding, User-Agent], x-content-type-options=[nosniff]}, hasValue=false} [Error: {"id":"20190712_134639_00013_8nfe8","infoUri":"https://presto-test.gett.io:8443/ui/query.html?20190712_134639_00013_8nfe8","partialCancelUri":"http://172.30.13.50:8889/v1/stage/20190712_134639_00013_8nfe8.0","nextUri":"https://presto-test.gett.io:8443/v1/statement/executing/20190712_134639_00013_8nfe8/x5ab6b8afff9f44a284f0fc7ae99013b6/1","columns":[{"name":"uuid","type":"uuid","typeSignature":{"rawType":"uuid","typeArguments":[],"literalArguments":[],"arguments":[]}}],"data":[["63a60097-202a-4705-9753-4c14f22288e0"]],"stats":{"state":"RUNNING","queued":false,"scheduled":true,"nodes":1,"totalSplits":17,"queuedSplits":0,"runningSplits":16,"completedSplits":1,"cpuTimeMillis":10,"wallTimeMillis":98,"queuedTimeMillis":0,"elapsedTimeMillis":387,"processedRows":1,"processedBytes":0,"peakMemoryBytes":0,"spilledBytes":0,"rootStage":{"stageId":"0","state":"RUNNING","done":false,"nodes":1,"totalSplits":16,"queuedSplits":0,"runningSplits":16,"completedSplits":0,"cpuTimeMillis":1,"wallTimeMillis":25,"processedRows":0,"processedBytes":0,"subStages":[{"stageId":"1","state":"FINISHED","done":true,"nodes":1,"totalSplits":1,"queuedSplits":0,"runningSplits":0,"completedSplits":1,"cpuTimeMillis":9,"wallTimeMillis":73,"processedRows":1,"processedBytes":0,"subStages":[]}]},"progressPercentage":5.882352941176471},"warnings":[]}
]
	at com.facebook.presto.jdbc.internal.client.StatementClientV1.requestFailedException(StatementClientV1.java:458)
	at com.facebook.presto.jdbc.internal.client.StatementClientV1.advance(StatementClientV1.java:397)
	at com.facebook.presto.jdbc.PrestoResultSet.getColumns(PrestoResultSet.java:1742)
	at com.facebook.presto.jdbc.PrestoResultSet.<init>(PrestoResultSet.java:119)
	at com.facebook.presto.jdbc.PrestoStatement.internalExecute(PrestoStatement.java:250)
	... 14 more
Caused by: java.lang.IllegalArgumentException: Unable to create class com.facebook.presto.jdbc.internal.client.QueryResults from JSON response:
[{"id":"20190712_134639_00013_8nfe8","infoUri":"https://presto-test.gett.io:8443/ui/query.html?20190712_134639_00013_8nfe8","partialCancelUri":"http://172.30.13.50:8889/v1/stage/20190712_134639_00013_8nfe8.0","nextUri":"https://presto-test.gett.io:8443/v1/statement/executing/20190712_134639_00013_8nfe8/x5ab6b8afff9f44a284f0fc7ae99013b6/1","columns":[{"name":"uuid","type":"uuid","typeSignature":{"rawType":"uuid","typeArguments":[],"literalArguments":[],"arguments":[]}}],"data":[["63a60097-202a-4705-9753-4c14f22288e0"]],"stats":{"state":"RUNNING","queued":false,"scheduled":true,"nodes":1,"totalSplits":17,"queuedSplits":0,"runningSplits":16,"completedSplits":1,"cpuTimeMillis":10,"wallTimeMillis":98,"queuedTimeMillis":0,"elapsedTimeMillis":387,"processedRows":1,"processedBytes":0,"peakMemoryBytes":0,"spilledBytes":0,"rootStage":{"stageId":"0","state":"RUNNING","done":false,"nodes":1,"totalSplits":16,"queuedSplits":0,"runningSplits":16,"completedSplits":0,"cpuTimeMillis":1,"wallTimeMillis":25,"processedRows":0,"processedBytes":0,"subStages":[{"stageId":"1","state":"FINISHED","done":true,"nodes":1,"totalSplits":1,"queuedSplits":0,"runningSplits":0,"completedSplits":1,"cpuTimeMillis":9,"wallTimeMillis":73,"processedRows":1,"processedBytes":0,"subStages":[]}]},"progressPercentage":5.882352941176471},"warnings":[]}
]
	at com.facebook.presto.jdbc.internal.client.JsonResponse.<init>(JsonResponse.java:70)
	at com.facebook.presto.jdbc.internal.client.JsonResponse.execute(JsonResponse.java:144)
	at com.facebook.presto.jdbc.internal.client.StatementClientV1.advance(StatementClientV1.java:383)
	... 17 more
Caused by: java.lang.IllegalArgumentException: Invalid JSON string for [simple type, class com.facebook.presto.jdbc.internal.client.QueryResults]
	at com.facebook.presto.jdbc.internal.airlift.json.JsonCodec.fromJson(JsonCodec.java:137)
	at com.facebook.presto.jdbc.internal.client.JsonResponse.<init>(JsonResponse.java:67)
	... 19 more
Caused by: com.facebook.presto.jdbc.internal.jackson.databind.exc.InvalidDefinitionException: Cannot construct instance of `com.facebook.presto.jdbc.internal.client.QueryResults`, problem: Illegal base64 character 2d
 at [Source: (String)"{"id":"20190712_134639_00013_8nfe8","infoUri":"https://presto-test.gett.io:8443/ui/query.html?20190712_134639_00013_8nfe8","partialCancelUri":"http://172.30.13.50:8889/v1/stage/20190712_134639_00013_8nfe8.0","nextUri":"https://presto-test.gett.io:8443/v1/statement/executing/20190712_134639_00013_8nfe8/x5ab6b8afff9f44a284f0fc7ae99013b6/1","columns":[{"name":"uuid","type":"uuid","typeSignature":{"rawType":"uuid","typeArguments":[],"literalArguments":[],"arguments":[]}}],"data":[["63a60097-202a-470"[truncated 827 chars]; line: 1, column: 1326]
	at com.facebook.presto.jdbc.internal.jackson.databind.exc.InvalidDefinitionException.from(InvalidDefinitionException.java:67)
	at com.facebook.presto.jdbc.internal.jackson.databind.DeserializationContext.instantiationException(DeserializationContext.java:1608)
	at com.facebook.presto.jdbc.internal.jackson.databind.deser.std.StdValueInstantiator.wrapAsJsonMappingException(StdValueInstantiator.java:484)
	at com.facebook.presto.jdbc.internal.jackson.databind.deser.std.StdValueInstantiator.rewrapCtorProblem(StdValueInstantiator.java:503)
	at com.facebook.presto.jdbc.internal.jackson.databind.deser.std.StdValueInstantiator.createFromObjectWith(StdValueInstantiator.java:285)
	at com.facebook.presto.jdbc.internal.jackson.databind.deser.ValueInstantiator.createFromObjectWith(ValueInstantiator.java:229)
	at com.facebook.presto.jdbc.internal.jackson.databind.deser.impl.PropertyBasedCreator.build(PropertyBasedCreator.java:195)
	at com.facebook.presto.jdbc.internal.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:488)
	at com.facebook.presto.jdbc.internal.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1287)
	at com.facebook.presto.jdbc.internal.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:326)
	at com.facebook.presto.jdbc.internal.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:159)
	at com.facebook.presto.jdbc.internal.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4013)
	at com.facebook.presto.jdbc.internal.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3042)
	at com.facebook.presto.jdbc.internal.airlift.json.JsonCodec.fromJson(JsonCodec.java:134)
	... 20 more
Caused by: java.lang.IllegalArgumentException: Illegal base64 character 2d
	at java.base/java.util.Base64$Decoder.decode0(Unknown Source)
	at java.base/java.util.Base64$Decoder.decode(Unknown Source)
	at java.base/java.util.Base64$Decoder.decode(Unknown Source)
	at com.facebook.presto.jdbc.internal.client.FixJsonDataUtils.fixValue(FixJsonDataUtils.java:184)
	at com.facebook.presto.jdbc.internal.client.FixJsonDataUtils.fixData(FixJsonDataUtils.java:78)
	at com.facebook.presto.jdbc.internal.client.QueryResults.<init>(QueryResults.java:70)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.facebook.presto.jdbc.internal.jackson.databind.introspect.AnnotatedConstructor.call(AnnotatedConstructor.java:124)
	at com.facebook.presto.jdbc.internal.jackson.databind.deser.std.StdValueInstantiator.createFromObjectWith(StdValueInstantiator.java:283)
	... 29 more
```

**Comment by guyco33** [comment]
PR https://github.com/dbeaver/dbeaver/pull/6265 solves this issue

**Comment by guyco33** [comment]
Fix is deployed in community release 6.1.3  by choosing PrestoSQL driver
